crates/telemetry/src/config.rs

```diff
@@ -0,0 +1,68 @@
+use envconfig::Envconfig;
+
+/// Provides configuration for the telemetry system.
+///
+/// Largely consists of OpenTelemetry variables defined
+/// [here](https://opentelemetry.io/docs/specs/otel/configuration/sdk-environment-variables/#general-sdk-configuration)
+/// but also has some Spin specific configuration.
+#[derive(Debug)]
+pub struct Config {
+    /// Disables OpenTelemetry for all signals (traces and metrics).
+    pub otel_sdk_disabled: bool,
```

Comment:
I don't love the double negative here (or the "SDK") but I assume that this is forced on us by OpenTelemetry naming conventions?

Reply:
Correct, this is forced on us by OTEL [naming conventions](https://opentelemetry.io/docs/specs/otel/configuration/sdk-environment-variables/#general-sdk-configuration).